src/main/java/com/google/flourbot/api/DriveCloudSheet.java

```diff
@@ -12,49 +12,66 @@
 import java.util.List;
 
 public class DriveCloudSheet implements CloudSheet {
-  Sheets sheetsService;
-  String spreadsheetId;
+  private String valueInputOption = "USER_ENTERED";
+  private String insertDataOption = "INSERT_ROWS";
+  private Sheets sheetsService;
+  private String spreadsheetId;
 
   public DriveCloudSheet(String documentId, Sheets documentService) {
     this.spreadsheetId = documentId;
     this.sheetsService = documentService;
   }
 
-  // TODO: replace hardcoded range with calculated range (so far values must be length 3)
-  public void appendRow(List<String> values) throws IOException, GeneralSecurityException {
+  private String getRange(int numEntryColumns, int numSkipColumns) {
+    // Returns range for sheets request by calculating the alphabetic name of
+    // the start and end column based on the number of columns filled and pre-skipped
+
+    String startColumn = this.toAlphabetic(numSkipColumns);
+    String endColumn = this.toAlphabetic(numSkipColumns + numEntryColumns - 1);
+    return "Sheet1!" + startColumn + "1:" + endColumn + "1";
+  }
+
+  private String toAlphabetic(int i) {
+    // Recursive strategy to build a string that represents the alphabetic name
+    // of a column based on column number i
+    // e.g. 0 -> A, 1 -> B... 25 -> Z, 26 -> AA...
     
-    // Generate request to append to sheet
-    String range = "Sheet1!A1:C1";
-    String valueInputOption = "USER_ENTERED";
-    String insertDataOption = "INSERT_ROWS";
+    // Only consider the absolute value of the number
+    if (i < 0) {
```

Comment:
https://en.wikipedia.org/wiki/Fail-fast

Input doesn't make sense? IllegalArgumentException, also Math.abs() is the right way of doing it.